drink/src/session/mocking_api.rs

```diff
@@ -33,20 +45,21 @@ where
             .expect("Should be able to acquire lock on registry")
             .salt();
 
+        let origin = T::convert_account_to_origin(T::default_actor());
         let mock_address = self
             .sandbox()
             .deploy_contract(
                 mock_bytes,
                 0u32.into(),
                 vec![],
-                salt,
-                T::default_actor(),
+                Some(salt),
+                origin,
```

Comment:
Something like the following should be doable:
```suggestion
                T::default_origin(),
```

Reply:
There is no method `default_origin` in the ink sandbox.